@bergercookie it's me or the failure of the build pipeline is related to other plugins, untouched by this PR?
```
============================= test session starts ==============================
platform linux -- Python 3.6.7, pytest-7.0.1, pluggy-0.12.0
rootdir: /home/travis/build/bergercookie/awesome-albert-plugins
collected 6 items                                                              
plugins/gmaps/gmaps-cli/tests/test_gmaps_cli.py ....s                    [ 83%]
plugins/saxophone/test/test_links.py F                                   [100%]
=================================== FAILURES ===================================
```
Could you check my code and eventually merge it anyway? I'm new to PRs on github, so I don't know if you can :smile:

--- a/plugins/emoji/README.md
+++ b/plugins/emoji/README.md
@@ -25,21 +25,13 @@ On additional key presses it allows for fuzzy search on the labels of each emoji
 
 ## Installation instructions
 
-### Requisites
+### Prerequisites
 
-#### Exec requisites
+#### Exec prerequisites
 
 - `xclip`: you must have installed xclip; ensure that xlip path is in PATH environment variable.
 
-_Example for Ubuntu_:
-
-```bash
-sudo apt-get install xclip
-which xclip  # i.e. /usr/bin/xclip
-export PATH="$PATH:/usr/bin/xclip"
-```
-
-##### Python requisites
+##### Python prerequisites
 
 - `em-keyboard`: [link to repository](https://github.com/hugovk/em-keyboard)
 - `fuzzywuzzy`

--- a/plugins/emoji/__init__.py
+++ b/plugins/emoji/__init__.py
@@ -13,7 +13,7 @@ from gi.repository import GdkPixbuf, Notify
 import pickle
 
 __title__ = "Emoji picker"
-__version__ = "0.4.1"
+__version__ = "0.4.0"
 __triggers__ = "em "
 __authors__ = "Nikos Koukis"
 __homepage__ = (

